### DISCUSSION TITLE
Daily E3SM-2-X picontrol wind stress fields publicly available?

### DISCUSSION THREAD

**Discussion by jacob-stu-allen** [E3SM-Project/E3SM#5524] [E3SM model help]
Hello!

I'm looking to modify wind stress fields in E3SM-2, but I will first need a long record (100 years +) of daily averaged wind stress from the model. I have access to NERSC machines and using pre-existing daily wind stress data will save us a simulation - is this available? 

Thanks in advance,
Jacob

**Comment by chengzhuzhang** [comment]
Hi @jacob-stu-allen Thanks for the inquiry. E3SM v2 runs are publicly available through NERSC HPSS and ESGF. I just looked at the data inventory. Unfortunately wind stress were not directly saved as daily output in the E3SM v2 production runs. However, the lowest model level meridional and zonal winds: UBOT and VBOT are available as daily output, which can be used to get estimated wind stress. 

Data access: https://portal.nersc.gov/archive/home/projects/e3sm/www/WaterCycle/E3SMv2/LR with accompanying documentation at https://e3sm-project.github.io/e3sm_data_docs. Another copy of  the native output is also available through the DOE Earth System Grid Federation (ESGF) at https://esgf-node.llnl.gov/search/e3sm/?model_version=2_0

**Reply by jacob-stu-allen** [reply]
Thank you very much!
